dln.c

```diff
@@ -447,6 +447,33 @@ dln_sym(void *handle, const char *symbol)
 }
 #endif
 
+void *
+dln_symbol(void *handle, const char *symbol)
+{
+#if defined(_WIN32) || defined(USE_DLN_DLOPEN)
+    if (EXTERNAL_PREFIX[0]) {
+        const size_t symlen = strlen(symbol);
+        char *const tmp = ALLOCA_N(char, symlen + sizeof(EXTERNAL_PREFIX));
+        if (!tmp) dln_memerror();
+        memcpy(tmp, EXTERNAL_PREFIX, sizeof(EXTERNAL_PREFIX) - 1);
+        memcpy(tmp + sizeof(EXTERNAL_PREFIX) - 1, symbol, symlen + 1);
+        symbol = tmp;
+    }
+    if (handle == NULL) {
+# if defined(USE_DLN_DLOPEN)
+        handle = dlopen(NULL, RTLD_FIRST);
+# else
+        // WIN32 seems not having similar feature to get the handle of the main process
```

Comment:
Is this true? I couldn't find any way to do that.

Reply:
`rb_libruby_handle()` should return the handle to `libruby`.
Rather `RTLD_FIRST` seems platform dependent.